plugins/backend/catalog-backend-module-okta/config.d.ts

```diff
@@ -22,7 +22,23 @@ export interface Config {
       /**
        * Okta configuration
        */
-      okta?: AccountConfig[];
```

Comment:
This import is now unused right?

Reply:
Seems like this introduced somewhere from main. I will remove it.